.github/workflows/windows_unit_tests.yaml

```diff
@@ -59,7 +59,18 @@ jobs:
           conda config --add channels conda-forge
           conda activate curr_py
           conda install numba -q -y
-      - name: Install EvalML with test requirements
+      - if: ${{ matrix.command == 'git-test-prophet' }}
+        name: Install EvalML with test requirements and prophet
+        shell: pwsh
+        run: |
+          . $env:USERPROFILE\Miniconda3\shell\condabin\conda-hook.ps1
+          conda activate curr_py
+          python -m pip install --upgrade pip
+          python -m pip install .[test]
+          python -m pip install .[prophet]
+          pip freeze
+      - if: ${{ matrix.command != 'git-test-prophet' }}
```

Comment:
For linux workflows, we also run `git-test-automl` with prophet installed. I decided not to do that for windows because of [known slowness](https://github.com/facebook/prophet/issues/122) when using the pip installation of prophet on windows - the tests were taking [over an hour](https://github.com/alteryx/evalml/actions/runs/4408390889/jobs/7723333291)